src/magic_folder/frontends/magic_folder.py

```diff
@@ -328,10 +328,6 @@ def save_magic_folders(node_directory, folders):
         yamlutil.safe_dump({u"magic-folders": folders}),
     )
 
-    config = configutil.get_config(os.path.join(node_directory, u"tahoe.cfg"))
-    configutil.set_config(config, "magic_folder", "enabled", "True")
-    configutil.write_config(os.path.join(node_directory, u"tahoe.cfg"), config)
-
```

Comment:
The `magic-folder list` tests invoke `magic-folder create` and other mutate-y commands.  Those commands all want to upgrade and rewrite the magic-folder configuration.  Previously this made sense but now the `[magic_folder]enabled` setting in `tahoe.cfg` is worse than useless.  We don't want to enable Tahoe-LAFS' built-in magic-folders support if we're running the stand-alone magic-folder.  So, the stand-alone magic-folder should never rewrite the Tahoe-LAFS config to turn on this toggle.